modules/tappxBidAdapter.js

```diff
@@ -262,11 +262,24 @@ function buildOneRequest(validBidRequests, bidderRequest) {
   imp.secure = 1;
 
   imp.bidfloor = utils.deepAccess(validBidRequests, 'params.bidfloor');
+  if (utils.isFn(validBidRequests.getFloor)) {
+    try {
+      imp.bidfloor = validBidRequests.getFloor({
+        currency: CUR,
+        mediaType: '*',
+        size: '*'
+      }).floor;
+    } catch (e) {
+      utils.logWarn(e);
```

Comment:
Could you add a prefix here so users are more easy to debug when this error is thrown:

utils.logWarn('tappx bid adapter: ', e);

Something like this.

Reply:
Hi!!!
I use finally use:
`utils.logWarn('[TAPPX]: ', e);`
to use the same prefix than my teammate.
Thanks for all!
(We made the push of this change)